extend-existing-app-theme-demo/pom.xml

```diff
@@ -0,0 +1,318 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <name>Reusing Theme jar Demo</name>
+    <groupId>com.vaadin.demo.apptheme</groupId>
+    <artifactId>extend-existing-app-theme-demo</artifactId>
+    <version>1.0-SNAPSHOT</version>
+    <packaging>jar</packaging>
+
+    <properties>
+        <java.version>11</java.version>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+
+        <vaadin.version>19.0-SNAPSHOT</vaadin.version>
```

Comment:
Now some modules use a snapshots, and some use alphas? Is it better to keep using alphas everywhere in the project?